src/telegram.ts

```diff
@@ -27,6 +28,12 @@ class Telegram extends ApiClient {
     } else if (fileId.file_path === undefined) {
       fileId = await this.getFile(fileId.file_id)
     }
+
+    // Local bot API instances return the absolute path to the file
+    if (fileId.file_path && isAbsolute(fileId.file_path)) {
+      return 'file://' + fileId.file_path
```

Comment:
In case Windows paths don't start with a slash:

```suggestion
      return 'file://' + fileId.file_path.replace(/^\//, '/')
```

Reply:
I don't understand. If it doesn't start with a slash, nothing will happen.

Reply:
One slash will be missing